### ISSUE TITLE
Documenation defaults to pre-release

### ISSUE THREAD

**Issue by dsolovay** [autofac/Documentation#16]
The documentation defaults to pre-release version, which is somewhat confusing to new users.   See http://stackoverflow.com/a/38796944/402949

I recommend having the current stable release be the default.

**Comment by tillig** [comment]
I understand the confusion, and I'm sorry, but I don't know if we can win on this for a couple of reasons:
- We only have one set of docs but something like 24 integration libraries. The docs only get one tag, which is generally the version of core Autofac... But as we add/update docs for the other libraries, that updated doc would be "hidden" if it didn't correspond to a new release tag.
- There's a non-trivial number of people looking for .NET Core info, especially since RTM, and that doc would be "hidden" until we're able to finish the last bits of the Autofac conversion.
- We get updates for docs - PRs for typos or corrections to examples - and those updates wouldn't be in the release tag. We would likely see PRs coming in for old docs for things that have already been updated or corrected.

Basically, if we made the default doc set a release tag, we'd fix the confusion for some but create confusion for others.

The pattern I've seen with ASP.NET Core and other recent docs is to have the active docs be the latest and allow folks to select a specific version if needed. That sort of jives with how we're doing it.

So... Good suggestion, but I think for now we need to keep the active doc the latest.

**Comment by dsolovay** [comment]
Thanks for explaining the issue, and the standards followed with ASP.NET
Core. Is it possible to have "latest" changed to "prerelease"?
Alternatively, it would be helpful to have a notice in the Getting Started
section, along these lines:

A Note on Versions
This documentation defaults to the current latest build, so some of the
syntax, features, and examples may not work with the current stable build
available on NuGet.  You can select the appropriate version of this
documentation using the widget in the bottom left of this screen.
